### DISCUSSION TITLE
Importing Uniswap V3 Core (solidity 0.7.6) to use in project with solidity version 0.8.15

### DISCUSSION THREAD

**Discussion by partylikeits1983** [foundry-rs/foundry#2297] [Help]
What's the best way to import smart contracts with an older version of solidity into a project with a newer pragma version in a foundry project?

**Comment by PaulRBerg** [comment]
This depends upon whether you want to import the v0.7 contract in the v>=0.8.0 contract, or you want to keep them separate. Could you clarify that?
